assets/scss/_variables_project.scss

```diff
@@ -6,6 +6,11 @@ Add styles or override variables from the theme here.
 
 $primary: #03153A;
 $secondary: #e1e5ee;
+$orange: #EE5007;
+$softer-orange: #FF7442;
```

Comment:
Why not orange, orangery and orangest? Wasted opportunity imho lol

Reply:
LOL ... I started with orange1 and orange2, then settled with this, but you have a point. I should have gone with organte and mar-a-lago-orange 🤣